app/views/shared/_bottom_cta.html.erb

```diff
@@ -2,6 +2,7 @@
   <p class="bottom-cta__heading"><%= heading %></p>
 
   <p class="bottom-cta__sub-heading"><%= subheading %></p>
-
-  <%= button %>
+  <% if current_user.nil? %>
```

Comment:
the bottom cta is on a lot of different pages, we wouldn't want this check running on those other pages.